### PR TITLE
Release 2.2.0 update changelog and docs

### PR DESCRIPTION
**Pull request by hlindberg**

### CODE REVIEW

**Comment by hlindberg** [comment]
Ping @nfagerlund

**Comment by ahpook** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+##2016-07-XX (TBD) - Release 2.2.0
+### Summary
+This release includes new features as well as bug fixes.
+
+The overview data model now contains more information. It used to only contain the top 10 nodes based on an issue count.
+Now the overview data model contains information about all nodes the overview is based on and with more details per node
```
Comment (jtappa) [inline CHANGELOG.md]: add some commas:

`Now, the overview data model contains information about all nodes the overview is based on, and`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+in the form of number of errors, warning, and catalog differences. This information is now used to better order the output
+as well as being easier to consume by custom post-processing and presentation logic.
+
+Attempting to use a node for which there was no available fact-set caused an error that did not indicate what the real problem was.
+Now, instead, deactivated or non existing nodes will be skipped. This supports the use case where a compilation is request
```
Comment (jtappa) [inline CHANGELOG.md]: This sentence is weird, "request" should be "requested" and the two uses "where" in the same sentence are confusing.

This supports the use case where a compilation is request where the list of nodes to compile for was generated from a list of certificates (which may contain deactivated nodes).

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+where the list of nodes to compile for was generated from a list of certificates (which may contain deactivated nodes).
+
+There were errors in the calculation and presentation of number of issues where the same problem could be present multiple times.
```
Comment (jtappa) [inline CHANGELOG.md]: missing an 'a', should be "of a number of issues"

**Comment by jtappa** [comment]
pinging @hlindberg for a few small edits.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
+##2016-07-XX (TBD) - Release 2.2.0
+### Summary
```
Comment (nfagerlund) [inline CHANGELOG.md]: MD formatting: for best renderer support, a header should be followed by a blank line.

File: CHANGELOG.md (hunk 2 above)
Comment (nfagerlund) [inline CHANGELOG.md]: or, "This makes it easier to request a compilation based on a list of node certificates, even if some of those nodes are deactivated."

**Comment by hlindberg** [comment]
Comments addressed.

File: lib/puppet_x/puppetlabs/preview/api/documentation/preview-help.md [hunk 5]
```diff
@@ -112,7 +112,8 @@ Note that all settings (such as 'log_level') affect both compilations.
   Specifies a file to read node-names from. If the file name is '-' file names are read
   from standard in. Each white-space separated sequence of characters is taken as a node name.
   This may be combined with additional nodes given on the command line. Duplicate entries (in given
-  file, or on command line) are skipped.
+  file, or on command line) are skipped. Deactivated and nodes for which there are no facts available
+  are skipped.
```
Comment (nfagerlund) [inline lib/puppet_x/puppetlabs/preview/api/documentation/preview-help.md]: 1. Change "may" to "can."
2. "Duplicated entries, deactivated nodes, and nodes with no facts available are skipped."

**Comment by hlindberg** [comment]
failing because of json_pure (separate issue).

**Pull request merged**